erpnext/assets/doctype/location/location_tree.js

```diff
@@ -27,6 +27,44 @@ frappe.treeview_settings["Location"] = {
 			condition: 'frappe.boot.user.can_create.indexOf("Location") !== -1'
 		}
 	],
+	onrender: function (node) {
+
+		if (!node.is_root) {
+			frappe.db.get_value("Location", node.data.value, "area")
+				.then((r) => {
+					// Place the location area on the right side, tree accounts style.
+					$('<span class="balance-area pull-right text-muted small">'
+						+ (`${((flt(r.message.area)).toLocaleString('en') + ' Sq m')}`)
```

Comment:
Why hard-coded "Sq m"? As we already have a dedicated field "Area UOM".

Reply:
Was an oversight, we are aware it needs to be translatable and will make those changes!

Reply:
We have made the changes expect a PR today.